### DISCUSSION TITLE
after calling whisper_full  takes much of time to product result

### DISCUSSION THREAD

**Discussion by Ehsanwww** [ggml-org/whisper.cpp#1585] [General]
Hello everyone, I use this great asset for godot 4 (Game Engine), I have issues when using real time speech to text of whisper, every time I call whisper_full   the result has no segment, I get data of mic every 10 seconds  and pass it into whisper_full 
@goncha 
@taf2 
@tmc

**Comment by cjheath** [comment]
> <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/20922281?s=64&amp;v=4">
> Ehsanwww

What part of "don't @ me" is difficult for you to understand. @ggerganov, please ban this guy.

**Reply by Ehsanwww** [reply]
oh, sorry, I delete that
